### PR TITLE
(WIP) Upgrade system namespace

### PR DESCRIPTION
**Pull request by MBishop17**

### CODE REVIEW

File: content/rancher/v2.x/en/upgrades/upgrades/namespace-migration/_index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+## Solution
+
+Reset the cluster nodes' network policies to restore connectivity.
+
+>**Prerequisites:**
+>
+>Download and setup [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+{{% tabs %}}
+{{% tab "Rancher Server Nodes" %}}
```
Comment (superseb) [inline content/rancher/v2.x/en/upgrades/upgrades/namespace-migration/_index.md]: This is not really applicable to Rancher Server Nodes, I would call it HA install probably

File: content/rancher/v2.x/en/upgrades/upgrades/namespace-migration/_index.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+1. From **Terminal**, change directories to your kubectl file that's generated during Rancher install, `kube_config_rancher-cluster.yml`. This file is usually in the directory where you ran RKE during Rancher installation.
+
+1. Before repairing networking, run the following two commands to make sure that your nodes are `Ready` and `Healthy`.
```
Comment (superseb) [inline content/rancher/v2.x/en/upgrades/upgrades/namespace-migration/_index.md]: That your nodes have status `Ready` and all the cluster components are `Healthy`

File: content/rancher/v2.x/en/upgrades/upgrades/namespace-migration/_index.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    kubectl get cs
+    NAME                 STATUS    MESSAGE              ERROR
+    scheduler            Healthy   ok
+    controller-manager   Healthy   ok
+    etcd-0               Healthy   {"health": "true"}
+    etcd-2               Healthy   {"health": "true"}
+    etcd-1               Healthy   {"health": "true"}
+    ```
+
+1. Check the `networkPolicy` for all clusters by running the following command. Point toward `kube_config_rancher-cluster.yml`.
```
Comment (superseb) [inline content/rancher/v2.x/en/upgrades/upgrades/namespace-migration/_index.md]: "Point toward `kube_config_rancher-cluster.yml`." should be included in the sentence or removed, it's already in the command and people have been notified of it's use in the first step.

File: content/rancher/v2.x/en/upgrades/upgrades/namespace-migration/_index.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+1. Check the `networkPolicy` for all clusters again to make sure the policies have a status of `false `.
+
+        kubectl --kubeconfig kube_config_rancher-cluster.yml get cluster -o=custom-columns=ID:.metadata.name,NAME:.spec.displayName,NETWORKPOLICY:.spec.enableNetworkPolicy
+        
+        ID        NAME      NETWORKPOLICY
+        c-59ptz   custom    false
+        local     local     false
+
+1. Now remove all `networkpolicies` from system namespaces.  Run this command for each cluster, using the kubeconfig generated by RKE.
```
Comment (superseb) [inline content/rancher/v2.x/en/upgrades/upgrades/namespace-migration/_index.md]: As this is separated into HA install and Rancher Launched Kubernetes, this is just run on the HA install.

**Comment by superseb** [comment]
Do we need to have the text in two places?

File: content/rancher/v2.x/en/upgrades/upgrades/namespace-migration/_index.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+>Download and setup [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+{{% tabs %}}
+{{% tab "Rancher Server Nodes" %}}
+1. From **Terminal**, change directories to your kubectl file that's generated during Rancher install, `kube_config_rancher-cluster.yml`. This file is usually in the directory where you ran RKE during Rancher installation.
+
+1. Before repairing networking, run the following two commands to make sure that your nodes are `Ready` and `Healthy`.
+
+    ```
+    kubectl get nodes
```
Comment (superseb) [inline content/rancher/v2.x/en/upgrades/upgrades/namespace-migration/_index.md]: Replace this block with the block from the installation docs:

```
kubectl get nodes

NAME                          STATUS    ROLES                      AGE       VERSION
165.227.114.63                Ready     controlplane,etcd,worker   11m       v1.10.1
165.227.116.167               Ready     controlplane,etcd,worker   11m       v1.10.1
165.227.127.226               Ready     controlplane,etcd,worker   11m       v1.10.1
```

**Comment by MBishop17** [comment]
> Do we need to have the text in two places?

I don't follow what this text is commenting on. What text are you referring to?

**Comment by MBishop17** [comment]
Closing in favor of #948

**Pull request closed without merging**